### ISSUE TITLE
Failure of "Send Order Email Copy" spams customers, every minute, forever.

### ISSUE THREAD

**Issue by lingwooc** [magento/magento2#17152]
Magento 2.2.5 resends emails to customers if the "Send Order Email Copy" feature fails. This is due to a naive check in Magento\Sales\Model\Order\Email\Sender.php . It was PR  magento/magento2#14051 to fix #13769 that did it. 

It doesn't discriminate between a failure in the email copy (probably to a company address) and an email to a customer. If the copy fails repeatedly (I've literally never had this feature work) then it will spam your customers once a minute for ever. This is _super_ dumb.

This is the code:
```
        try {
            $sender->send();
            $sender->sendCopyTo();
        } catch (\Exception $e) {
            $this->logger->error($e->getMessage());

            return false;
        }

        return true;
```

It should probably be:
```
        try {
            $sender->send();
        } catch (\Exception $e) {
                $this->logger->error($e->getMessage());
            return false;
        }
        try {
            $sender->sendCopyTo();
        } catch (\Exception $e) {
            $this->logger->error($e->getMessage());
        }
        return true;
```
Although really its a great demonstration of why using a boolean to determine an error state isn't a great idea.

### Preconditions
Magento 2.2.5
Broken "Send Order Email Copy"
Asyncronous email sending

### Steps to reproduce
1. Set magento2 to send a separate email copy of an order
2. Turn on "Asyncronous email sending"
3. Make an order

### Expected result

1. Set magento2 to send a separate email copy of an order
2. Turn on "Asyncronous email sending"
3. Make an order
4 Don't spam customers

### Actual result

1. Set magento2 to send a seperate email copy of an order
2. Turn on "Asyncronous email sending"
3. Make an order
4. Spam your customers

**Comment by magento-engcom-team** [comment]
Hi @lingwooc. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me {$VERSION} instance` 

where `{$VERSION}` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (2.2-develop +). 
For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation.

@lingwooc do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by engcom-backlog-nickolas** [comment]
Hello @lingwooc, thank you for your report.
We've acknowledged the issue and added to our backlog.

**Comment by sambolek** [comment]
#mm18hr

**Comment by slavvka** [comment]
Hi @lingwooc. Thank you for your report. 
The issue has been fixed in magento/magento2#18288 by @sambolek in 2.3-develop branch
Related commit(s):
 - [5548c9d06c74e0dc4d50742363518a3f6a7506d3](https://github.com/magento/magento2/commit/5548c9d06c74e0dc4d50742363518a3f6a7506d3)
 - [8c8b0a83af7fe1021b55c390f12dfd923eb3d1c7](https://github.com/magento/magento2/commit/8c8b0a83af7fe1021b55c390f12dfd923eb3d1c7)
 - [6fe930320a612def6b5dcfb6b950ed0785c5756c](https://github.com/magento/magento2/commit/6fe930320a612def6b5dcfb6b950ed0785c5756c)

The fix will be available with the upcoming 2.3.1 release.

**Comment by magento-engcom-team** [comment]
Hi @lingwooc. Thank you for your report. 
The issue has been fixed in magento/magento2#18376 by @sambolek in 2.2-develop branch
Related commit(s):
 - [058c254f0d39a6ab1cad94de42863117404cdfb5](https://github.com/magento/magento2/commit/058c254f0d39a6ab1cad94de42863117404cdfb5)

The fix will be available with the upcoming 2.2.8 release.
